src/panels/developer-tools/event/developer-tools-event.js

```diff
@@ -51,6 +51,10 @@ class HaPanelDevEvent extends EventsMixin(LocalizeMixin(PolymerElement)) {
           max-width: 800px;
           margin: 16px auto;
         }
+
+        a {
+          color: var(--dark-primary-color);
```

Comment:
We use primary color for links

Reply:
Updated (and on events-list which was also using dark